libc/include/llvm-libc-types/struct_epoll_event.h

```diff
@@ -11,7 +11,11 @@
 
 #include <llvm-libc-types/struct_epoll_data.h>
```

Comment:
I think this needs to be adjusted to "struct_epoll_data.h", based on the build failures.
https://buildkite.com/llvm-project/github-pull-requests/builds/46591#018e348b-db56-4d69-a0a2-ca7d6daf9f76